### DISCUSSION TITLE
I'm trying to develop a mobile navigation synchronization to watch applet，but...

### DISCUSSION THREAD

**Discussion by vaas1993** [zepp-health/zeppos-forum#97] [Developer Feedback]
My idea is to read the mobile phone status bar to get the status bar message sent by the mobile phone navigation app and sync it to the watch display. I read the Zepp OS document and found no API that can support me to read the mobile phone status bar message. I know this may involve privacy and security issues. Is there any feasible solution?

**Comment by MyoungXUE** [comment]
Hi, this is a very good suggestion, but I'm sorry, as far as I know there are no plans in the short term, mainly due to limited R&D resources. I'll send this idea back to the relevant colleagues.
